x-pack/plugins/security_solution/public/value_list/components/value_list_modal.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState, useCallback } from 'react';
+import { css } from '@emotion/css';
+import {
+  EuiModal,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiFlexItem,
+  EuiFlexGroup,
+  useEuiPaddingSize,
+  EuiText,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  EuiSearchBar,
+} from '@elastic/eui';
+import { useFindListItems, useGetListById } from '@kbn/securitysolution-list-hooks';
+import { FormattedDate } from '../../common/components/formatted_date';
+import { useKibana } from '../../common/lib/kibana';
+import { AddListItemPopover } from './add_list_item_popover';
+import { UploadListItem } from './upload_list_item';
+import { ListItemTable } from './list_item_table';
+import { Info } from './info';
+import type { SortFields, ValueListModalProps, OnTableChange, Sorting } from '../types';
+import {
+  INFO_UPDATED_AT,
+  INFO_TYPE,
+  INFO_UPDATED_BY,
+  INFO_TOTAL_ITEMS,
+  getInfoTotalItems,
+} from '../translations';
+
+const modalWindow = css`
+  min-height: 90vh;
+  margin-top: 5vh;
+  max-width: 1400px;
```

Comment:
I don't know how we're supposed to go about designating sizes for these kinds of things, but this seems somewhat arbitrary. Have you discussed these values with design?

Reply:
checked with @ARWNightingale, looks like it's fine